ocs_ci/helpers/dr_helpers_ui.py

```
@@ -1838,3 +1838,879 @@ def navigate_using_fleet_virtualization(acm_obj):
+    """
+    if expected_actions is None:
+        expected_actions = [
+            "Edit configuration",
+            "Failover",
+            "Relocate",
+            "Manage disaster recovery",
+        ]
+
+    acm_loc = locators_for_current_ocp_version()["acm_page"]
+
+    log.info(f"Verifying kebab menu actions for app: {app_name}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)